data/templates/Detector_HostingEnvironment.csx

```diff
@@ -0,0 +1,19 @@
+private static string GetQuery(OperationContext<HostingEnvironment> cxt)
+{
+    return
+    $@"<YOUR_TABLE_NAME>
```

Comment:
This may be an annoying ask, but what do you think about making the template queries in the form of something like

`
let TenantList = {Utilities.getTenantList(cxt.Resource)}
let startTime = {cxt.StartTime}
let endTime = {cxt.EndTime}
Table | where TIMESTAMP > startTime and TIMESTAMP < endTime and Tenant in TenantList
| rest of query

I was thinking about this recently, and it seems to me that this would be easier and more intuitive for the user to transfer from kusto. Also it seems like most other people on the team use let statements for variables in their queries as standard practice.

Reply:
see these improvements can be done separately also. Doesnt have to be part of filters work.